### PR TITLE
Python 3 support complete, so removed warning

### PR DESCRIPTION
**Pull request by Fenn-CS**
`Python3` porting is complete so users should not be warned when running on a `python3 venv`

REQUIRED FOR : [`Tahoe-LAFS 1.16.0` Release](https://github.com/tahoe-lafs/tahoe-lafs/pull/1113)

Closes : https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3781

### CODE REVIEW

**Comment by itamarst** [comment]
Hm. On the one hand, it's done. On the other hand, probably worth emphasizing it's beta status and the project really wants users' feedback?

**Comment by exarkun** [comment]
> Hm. On the one hand, it's done. On the other hand, probably worth emphasizing it's beta status and the project really wants users' feedback?

At this point in the effort, though, maybe the README or the release announcement would be a better place to emphasize this?

**Comment by exarkun** [comment]
There are two failed jobs on CircleCI.  Since CircleCI built for the fork instead of the main repo, I can't make it retry them.  One of them looks intermittent and unrelated for sure.  The other I'm not sure about.  It would be nice to see these rerun.

**Comment by Fenn-CS** [comment]
I would look into those failures.

**Comment by itamarst** [comment]
> At this point in the effort, though, maybe the README or the release announcement would be a better place to emphasize this?

Sure I don't have a strong opinion, as long as it's mentioned somewhere.

**Comment by Fenn-CS** [comment]
> There are two failed jobs on CircleCI. Since CircleCI built for the fork instead of the main repo, I can't make it retry them. One of them looks intermittent and unrelated for sure. The other I'm not sure about. It would be nice to see these rerun.

I did an earlier rerun. Which failed again but I ran locally and found no issues, nonetheless I have fixed conflicts with updates to master and is observing one more time.

**Comment by Fenn-CS** [comment]
Everything looks fine now. Where exactly, would anyone suggest we transfer the notice to? The infant-support of Python 3 was [mentioned](https://github.com/tahoe-lafs/tahoe-lafs/pull/1113/commits/6e3a90af9b1a946745bafd3fcc03913d7cf72bfd) in the release draft. Perhaps we might add some emphasis there or someone else?

**Comment by itamarst** [comment]
Maybe in the documentation as well?

**Comment by meejah** [comment]
Yeah, I think mentioning in .. README? .. or similar would be good. Something to the effect that 1.16.x is the first "Python3 and Python2" supporting release. I guess the thing we'd want people to do with that knowledge is a) [still] file bugs found and b) prepare for the lack of "python2" in the future?

**Comment by itamarst** [comment]
That's fine with me.

File: README.rst [hunk 1]
```diff
@@ -56,6 +56,11 @@ For more detailed instructions, read `docs/INSTALL.rst <docs/INSTALL.rst>`__ .
 
 Once tahoe --version works, see `docs/running.rst <docs/running.rst>`__ to learn how to set up your first Tahoe-LAFS node.
 
+🐍 Python 3 Support 
+--------------------
+
+Python 3 support has been introduced starting with Tahoe-LAFS 1.16.0, alongside Python 2. System admnistrators are advised to start running Tahoe on Python 3 and should expect Python 2 support to be dropped in a future version.  Please, feel free to file issues if you run into bugs while running Tahoe on Python 3.
```
Comment (exarkun) [inline README.rst:62]:
```suggestion
Python 3 support has been introduced starting with Tahoe-LAFS 1.16.0, alongside Python 2.
System administrators are advised to start running Tahoe on Python 3 and should expect Python 2 support to be dropped in a future version.
Please, feel free to file issues if you run into bugs while running Tahoe on Python 3.
```

**Review by exarkun — APPROVED** [review]
Thanks.  One formatting suggestion change for README.rst then I think it is good to merge.

**Pull request merged**